docs/modules/ROOT/workflows/router_workflow.puml

```diff
@@ -0,0 +1,93 @@
+' Copyright 2019 The Alcor Authors.
+
+' Licensed under the Apache License, Version 2.0 (the "License");
+'        you may not use this file except in compliance with the License.
+'        You may obtain a copy of the License at
+
+'        http://www.apache.org/licenses/LICENSE-2.0
+
+'        Unless required by applicable law or agreed to in writing, software
+'        distributed under the License is distributed on an "AS IS" BASIS,
+'        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+'        See the License for the specific language governing permissions and
+'        limitations under the License.
+
+@startuml
+
+skinparam monochrome true
+
+collections "Port Manager"
+collections "Data Plane Manager"
+
+box "Compute Host 1" #LightBlue
+participant "ACA on \nHost 1"
+end box
+
+box "Compute Host 2" #LightGreen
+participant "ACA on \nHost 2"
+end box
+
+==All Scenarios: Alcor DVR created with both GREEN and RED network gateway connected==
+
+==Scenario 0: Alcor DVR creation flow ==
```

Comment:
The workflow looks fine between DPM and ACA. But DPM and above needs to be finalized by @haboy52581 later.